pkg/skaffold/schema/latest/config.go

```diff
@@ -177,11 +199,15 @@ type KubectlFlags struct {
 
 // HelmDeploy contains the configuration needed for deploying with helm
 type HelmDeploy struct {
+	Deployer `yaml:",inline"`
```

Comment:
You don't need a new struct - you can just add `Context` to the DeployConfig since all deployers will be forced to respect it.